keras/backend/theano_backend.py

```diff
@@ -1054,11 +1054,15 @@ def deconv2d(x, kernel, output_shape, strides=(1, 1),
     np_kernel = kernel.eval()
     filter_shape = _preprocess_filter_shape(dim_ordering, shape(kernel))
     
+    _t = ()
     for v in output_shape:
         try:
-            v = v.eval()
+            _v = v.eval()
```

Comment:
Here I stuck probably the same issue that @yaringal mentioned - if we infer output_shape, it becomes the tuple of Theano variables/expressions, which Theano couldn't accept as an input due to some reasons.
So I try to evaluate them, but here I get another error:

> MissingInputError: An input of the graph, used to compute Shape(input_25), was not provided and not given a value.Use the Theano flag exception_verbosity='high',for more information on this error.

I'm not an expert in Theano, so have no idea how to fix it quick.